server/src/main/java/io/druid/server/ClientQuerySegmentWalker.java

```diff
@@ -84,25 +82,13 @@ public ClientQuerySegmentWalker(
     final FinalizeResultsQueryRunner<T> baseRunner = new FinalizeResultsQueryRunner<T>(
         toolChest.postMergeQueryDecoration(
             toolChest.mergeResults(
-                new MetricsEmittingQueryRunner<T>(
```

Comment:
I agree that we are not losing too much information, but for systems that rely on existing metrics to be present that can be an issue. Either we should notify users of backwards incompatible changes, or try to maintain backwards compatibility whenever possible.